src/raster_footprint/cli.py

```diff
@@ -0,0 +1,132 @@
+# ruff: noqa: E501
```

Comment:
I prefer to put these in `pyproject.toml`, e.g.:

```toml
[tool.ruff.per-file-ignores]
"src/raster_footprint/cli.py" = ["E501"]
```

Reply:
🍬 I like. Done.